Library/Homebrew/language/python.rb

```diff
@@ -366,7 +366,12 @@ def pip_install(targets, build_isolation: true)
           targets = Array(targets)
           targets.each do |t|
             if t.is_a?(Resource)
-              t.stage { do_install(Pathname.pwd, build_isolation:) }
+              # We support wheel resources, but only when they're marked as pure Python.
+              if t.url.end_with? "-none-any.whl"
+                do_install(t.fetch, build_isolation:, allow_wheels: true)
```

Comment:
@Bo98 LMK what you think of this -- `Downloadable#fetch` is the API I'm familiar with, but there might be a cleaner way to do this.

Reply:
`stage` usually does `fetch unless downloaded?` under the hood.